src/rate_limit.ts

```diff
@@ -0,0 +1,81 @@
+import {DateTime} from "luxon";
+import {ioc} from "@nicolawealth/ioc";
+
+const nowMs = (): number => DateTime.now().toUTC().toMillis();
```

Comment:
We can actually cut out the dependency on Luxon by simplifying this down to `Date.now()`. Timezone correctness is not really important here because all we care about is the difference between 2 timestamps.